kubevirtci

```diff
@@ -171,6 +171,15 @@ function kubevirtci::create_cluster() {
 	echo "Using cluster template $template"
 
 	$CLUSTERCTL_PATH generate cluster ${TENANT_CLUSTER_NAME} --target-namespace ${TENANT_CLUSTER_NAMESPACE} --kubernetes-version ${CAPK_GUEST_K8S_VERSION} --control-plane-machine-count=1 --worker-machine-count=1 --from $template | ${_kubectl} apply -f -
+
+	echo "Wait for tenant cluster to be ready"
+	${_kubectl} wait cluster -n kvcluster kvcluster --for=condition=Ready --timeout=5m
+
+	echo "Wait for tenant cluster kubernetes apiserver up"
+	kubevirtci::retry_until_success kubevirtci::kubectl_tenant get pods -n kube-system
+
+	echo "Waiting for worker VM in tenant cluster namespace"
+	kubevirtci::wait_for_vm "${TENANT_CLUSTER_NAME}-md-"
```

Comment:
can we get some time outs for these? Below is one way to kind of perform rough timeouts using bash.

```
seconds=0
until $@; do
    ((seconds++)) && ((seconds == 30)) && echo "condition not met" && exit 1
    echo "waiting for \"$@\""
    sleep 1
done
```